src/client.js

```diff
@@ -79,12 +81,15 @@ class Client extends events {
 			disconnectReason = data;
 		});
 
-		socket.on("disconnect", () => {
+		socket.on("disconnect", (data) => {
+			wsReason = data;
```

Comment:
Why

Reply:
to log the disconnect reason provided by skribbl.io